pkg/rpc/wrappers/block.go

```diff
@@ -1,34 +1,102 @@
 package wrappers
 
 import (
+	"strconv"
+
 	"github.com/CityOfZion/neo-go/pkg/core"
 	"github.com/CityOfZion/neo-go/pkg/core/block"
+	"github.com/CityOfZion/neo-go/pkg/core/transaction"
+	"github.com/CityOfZion/neo-go/pkg/io"
 	"github.com/CityOfZion/neo-go/pkg/util"
 )
 
 type (
+	// Tx wrapper used for the representation of
+	// transaction on the RPC Server.
+	Tx struct {
+		TxID       util.Uint256            `json:"txid"`
+		Size       int                     `json:"size"`
+		Type       string                  `json:"type"`
+		Version    uint8                   `json:"version"`
+		Attributes []transaction.Attribute `json:"attributes"`
+		VIn        []transaction.Input     `json:"vin"`
+		VOut       []transaction.Output    `json:"vout"`
+		Scripts    []transaction.Witness   `json:"scripts"`
+
+		SysFee string `json:"sys_fee"`
+		NetFee string `json:"net_fee"`
+
+		Nonce uint32 `json:"nonce,omitempty"`
+	}
+
 	// Block wrapper used for the representation of
 	// block.Block / block.Base on the RPC Server.
 	Block struct {
-		*block.Block
-		Confirmations uint32       `json:"confirmations"`
-		NextBlockHash util.Uint256 `json:"nextblockhash,omitempty"`
-		Hash          util.Uint256 `json:"hash"`
+		Hash              util.Uint256 `json:"hash"`
+		Size              int          `json:"size"`
+		Version           uint32       `json:"version"`
+		NextBlockHash     util.Uint256 `json:"nextblockhash,omitempty"`
```

Comment:
`omitempty` is not needed anymore?

Reply:
NextBlockHash should be set only if it's exists